### ISSUE TITLE
Sound Isn't Working

### ISSUE THREAD

**Issue by greenDarkness** [dylmeadows/EonTimer#6]
The latest release of EonTimer doesn't have sound for me. I've tried switching between Audio, and A/V mode, as well as cycling through Beep, Ding, Pop, and Tick. 
The sound works fine for me in the beta release, but not at all in this version.

The way that I am running the program is as follows:
Extract the two folders from the zip, and place them in an EonTimer folder.
Open the bin folder, and double click "EonTimer.bat" to launch.
Then click start, and no sound will play.

I'm running Windows 7, Service Pack 1, and it's a 64 bit computer.

Thanks for taking the time to read this and for all your work with eon timer.

**Comment by dylmeadows** [comment]
Thanks for taking the time to submit the issue. Let me spin up a Windows 7 VM and give it a test. Hopefully the error will be pretty apparent. I'll follow up with you once I've learned more.

**Comment by dylmeadows** [comment]
I ran into a different problem trying to start the application in Windows 7. I think both problems might be caused by how windows handles file paths. I'll get started trying to resolve that.

**Comment by dylmeadows** [comment]
@greenDarkness is there anyway that you can run EonTimer.bat from the Command Line and see if there are any errors displaying? I tried on a clean install of Windows 7 and I wasn't running into any problems.

**Comment by greenDarkness** [comment]
The file path I am running from is a bit long. I can try running it from desktop, and then get back to you.

Yeah, I can run it from the command line. Do I just need to input the code from EonTimer.bat into the console?

**Comment by dylmeadows** [comment]
Let me test with those conditions specifically. It might be an issue with you running an earlier version of Java 8. I should be able to confirm that with a VM though.

**Comment by greenDarkness** [comment]
Sounds good. Hopefully that's the case, and I'll just update to fix the problem.

**Comment by dylmeadows** [comment]
@greenDarkness was able to confirm that the sound wasn't working due to running an older version of Java. Whenever I ran it with 8.0.111 (the one you have installed) the sound wasn't working. I uninstalled that version and installed 8.0.211 (the most recent) and sound was working as expected. Here you go:

https://www.oracle.com/technetwork/java/javase/downloads/jre8-downloads-2133155.html

Going to close this now since it's resolved! :) Thanks for the report!
